src/mlagility/api/script_api.py

```diff
@@ -8,6 +8,7 @@
 from groqflow.justgroqit.stage import Sequence
 import mlagility.cli.slurm as slurm
 import mlagility.common.filesystem as filesystem
+import mlagility.common.labels as lab
```

Comment:
```suggestion
import mlagility.common.labels as labels_library
```

Style: don't use uncommon abbreviations. If you're unable to use `labels` due to a namespace collision, `labels_library` is a reasonable name.

Reply:
Accepted suggestion.